cookomatic_api/tests/api/test_dish.py

```diff
@@ -26,17 +44,18 @@ def create_app(self):
         return app
 
     def test_get_dish(self):
-        dish_id = Dish(name='Dish 1').put().id()
-        expected = {'name': 'Dish 1', 'steps': []}
+        dish_id = Dish(**self.SAMPLE_DISH).put().id()
+        expected = self.SAMPLE_DISH
 
         response = self.client.get('/v1/dish/%s' % dish_id, content_type='application/json')
 
         self.assertEqual(expected, response.json)
```

Comment:
[This test[(https://travis-ci.org/cookomatic/cookomatic-api/jobs/206493462#L615) shows that it's trying to compare a JSON response (`response.json`) with the `SAMPLE_DISH` variable. Looks like one of them is unicode encoded and the other is not ... weird. Your `expected` variable will need to have the deserialized values as well to compare to.